src/lib.rs

```diff
@@ -145,6 +145,8 @@
 //!
 //! Wry uses a set of feature flags to toggle several advanced features.
 //!
+//! - `native` (default): Enables native WebView framework on the platform. This must be enabled
+//! for the crate to work. This feature is in preparasion of other ports like cef and servo.
```

Comment:
```suggestion
//! for the crate to work. This feature was added in preparation for other ports like cef and servo.
```
Only wanted to fix the preparation typo, so if you're not happy with the new wording we can change it back again ;)

Reply:
Thanks! I'll fix it